--- services/surfaceflinger/BufferLayer.cpp.Original	2019-05-30 12:04:17.498290116 +0100
+++ services/surfaceflinger/BufferLayer.cpp	2019-05-30 12:05:04.134817751 +0100
@@ -633,7 +633,13 @@
         visible.dump(LOG_TAG);
     }
 
-    error = hwcLayer->setSurfaceDamage(surfaceDamageRegion);
+    if(mFlinger->mDamageUsesScreenReference) {
+        const auto& frame = hwcInfo.displayFrame;
+        auto fullSource = Region(Rect(frame.left, frame.top, frame.right, frame.bottom));
+        error = hwcLayer->setSurfaceDamage(fullSource);
+    } else {
+        error = hwcLayer->setSurfaceDamage(surfaceDamageRegion);
+    }
     if (error != HWC2::Error::None) {
         ALOGE("[%s] Failed to set surface damage: %s (%d)", mName.string(),
               to_string(error).c_str(), static_cast<int32_t>(error));
